app/controllers/plans_controller.rb

```diff
@@ -43,4 +43,21 @@ def search
       # TODO
     end
   end
+
+  def set_autofinger_subscription
+    owner = current_account
+    interest = Account.find_by_username(params[:id])
+    autofinger = owner.interests_in_others.find_or_create_by_interest(interest.id)
+    success = autofinger.update_attributes(priority: params[:priority])
+    if success
+      if params[:priority] == '0'
+        notice = 'User was removed from your autoread list.'
+      else
+        notice = "User is now on your autoread list with priority level of #{params[:priority]}."
+      end
+    else
+      notice = "Could not change autoread priority. Please try again."
```

Comment:
Really here, I think just "Could not change autoread priority" is sufficient.  Trying again is probably not going to fix anything that did go wrong and it's unlikely to get there, so that way it at least it is not giving someone the wrong impression about a fix.  It's a small detail and not worth a whole lot of time if you have more interesting things to do.